presets/TARGET_DISCO_F769NI/TARGET_BL_INTERNAL_FLASH/mbed_app.json

```diff
@@ -2,39 +2,42 @@
-                    "+--------------------------+ <-+ mbed-bootloader.application-start-address",
-                    "|Active App Metadata Header|",
-                    "+--------------------------+ <-+ update-client.application-details",
-                    "|                          |",
-                    "|        Bootloader        |",
-                    "|                          |",
-                    "+--------------------------+ <-+ 0",
-                    "",
-                    "",
-                    "NRF52840_DK: setting `target.static_memory_defines` as `true` is an alternative for the workaround",
-                    "https://github.com/ARMmbed/mbed-os/pull/11510 . The PR in question is waiting for approval"
-            ]
-        }
+                "This configuration takes advantage of the fact that most MCUs have 2 banks of flash",
+                "which can be read from and written to independently of each other. Hence we put all",
+                "executables on the first bank, and all storage on the second bank such that reading",
+                "and writing storage does not halt execution. The regions are as follows:",
+                "1. Bootloader - 32kb from the beginning of flash",
+                "2. Active App Metadata Header - 512B from the end of Bootloader",
+                "3. Active App - From end of header to the end of first bank of flash",
+                "4. TDB internal storage - 32kb from the beginning of the second bank of the flash",
+                "5. Firmware Candidate Storage - from the end of the TDB internal storage",
+                "+--------------------------+",
+                "|                          |",
+                "|                          |",
+                "|                          |",
+                "|Firmware Candidate Storage|",
+                "|                          |",
+                "|                          |",
+                "|                          |",
+                "|--------------------------| <-+ fota.storage-start-address",
+                "|                          |",
+                "|    storage_tdb_internal  |",
+                "|                          |",
+                "+--------------------------+ <-+storage_tdb_internal.internal_base_address",
+                "|                          |",
+                "|        Active App        |",
+                "|                          |",
+                "+--------------------------+ <-+ mbed-bootloader.application-start-address",
+                "|Active App Metadata Header|",
+                "+--------------------------+ <-+ mbed-bootloader.application-header-address",
+                "|                          |",
+                "|        Bootloader        |",
+                "|                          |",
+                "+--------------------------+ <-+ 0",
+                "",
+                ""
+        ]
+    }
```

Comment:
something is wrong with the indentation here
Where was the tdb section in the previous configuration?
Or was it just not well documented?

Reply:
In previous configuration TDB section wasn't mentioned.
I think the indentation now is correct, there was an extra tab before that.